config/v1/types_infrastructure.go

```diff
@@ -196,14 +196,6 @@ type BareMetalPlatformStatus struct {
 	// ingressIP is an external IP which routes to the default ingress controller.
 	// The IP is a suitable target of a wildcard DNS record used to resolve default route host names.
 	IngressIP string `json:"ingressIP,omitempty"`
-
-	// nodeDNSIP is the IP address for the internal DNS used by the
-	// nodes. Unlike the one managed by the DNS operator, `NodeDNSIP`
-	// provides name resolution for the nodes themselves. There is no DNS-as-a-service for
-	// BareMetal deployments. In order to minimize necessary changes to the
-	// datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames
-	// to the nodes in the cluster.
-	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
```

Comment:
When and where this was deprecated, why that information was not exposed to consumers of the API before?

Reply:
Good point, we never actually deprecated this. It was just discovered that we no longer need it thanks to some other changes. For the moment I guess we should deprecate it and leave it in the api. It won't be used by anything, but at least it won't break any consumers.

What is the process for deprecating a field like this?